src/playwright-api/makeTest.ts

```diff
@@ -5,6 +5,7 @@ import type {
   PlaywrightWorkerArgs,
   PlaywrightWorkerOptions,
 } from '@playwright/test';
+import type { ChromaticConfig, ChromaticStorybookParameters } from 'src/types';
```

Comment:
Does this work?

```suggestion
import type { ChromaticConfig, ChromaticStorybookParameters } from '../../types';
```

Reply:
@skitterm it did! 767add76848d30242bba83d08f9c08d2b35f285d